.github/CODEOWNERS

```diff
@@ -43,3 +43,11 @@
  
 # ServiceLabel: %area-KeyVault
 # ServiceOwners:                       @conniey @JonathanCrd
+
+# PRLabel: %area-ServiceBus
+/src/Arguments/ServiceBus/             @conniey @JonathanCrd
+/src/Commands/ServiceBus/              @conniey @JonathanCrd
+/src/Services/Azure/ServiceBus/        @conniey @JonathanCrd
+ 
+# ServiceLabel: %area-ServiceBus
+# ServiceOwners:                       @conniey @JonathanCrd
```

Comment:
Should be @shankarsama @EldertGrootenboer from the .net SDK codeowners entry for serviceowners, I believe